src/Filters/DataAccessor.php

```diff
@@ -39,6 +39,16 @@ class DataAccessor
      */
     private $attributes;
 
+    /**
+     * @var array
+     */
+    private $features;
+
+    /**
+     * @var array
+     */
+    private $featureValues;
```

Comment:
```suggestion
    private $featureValues = [];
```
just to avoid possibility of multiple types? (cuz by default its null)